### DISCUSSION TITLE
How do I support .NETFramework4.5.1

### DISCUSSION THREAD

**Discussion by HarlanGitHub** [cefsharp/CefSharp#3948] [General]
I need your help with a problem：

We have a client software system developed using .NETFramework4.5.1

Now I want to integrate Cefsharp, but I find that the latest version of Cefsharp cannot be installed.

How do I get support?   Please tell me,thanks.

**Comment by amaitland** [comment]
You would need to modify and build CefSharp from source. 

Honestly I'd suggest looking at other browser options to see if any meet your requirements before going down this path.
